ledger/src/blockstore.rs

```diff
@@ -4316,49 +4316,49 @@ fn adjust_ulimit_nofile(_enforce_ulimit_nofile: bool) -> Result<()> {
 }
 
 #[cfg(unix)]
-fn adjust_ulimit_nofile(enforce_ulimit_nofile: bool) -> Result<()> {
+fn adjust_ulimit_nofile(_enforce_ulimit_nofile: bool) -> Result<()> {
     // Rocks DB likes to have many open files.  The default open file descriptor limit is
     // usually not enough
     // AppendVecs and disk Account Index are also heavy users of mmapped files.
     // This should be kept in sync with published validator instructions.
     // https://docs.solana.com/running-validator/validator-start#increased-memory-mapped-files-limit
-    let desired_nofile = 1_000_000;
```

Comment:
Yes, thanks for noticing.  I comment out most of that file when I run local tests because I couldn't figure out how to prevent this code from failing while I was running tests.  When I restored the code I must have made a mistake and didn't notice before checking in.  Will fix, thanks for noticing!